code/game/machinery/doors/door.dm

```diff
@@ -27,6 +27,10 @@
 	var/visible = TRUE
 	var/operating = FALSE
 	var/glass = FALSE
+	/// Do we need to keep track of a filler panel with the airlock
+	var/multi_tile
+	/// A filler object used to fill the space of multi-tile airlocks
+	var/obj/structure/fluff/airlock_filler/filler
```

Comment:
Similarly here register to the qdeleting signal of the filler and clear this reference just in case something deletes it unexpectedly (optionally, kill this too if something unexpectedly deletes the filler)